### DISCUSSION TITLE
Cannot Subscribe more than 50 topics

### DISCUSSION THREAD

**Discussion by altatechmark** [aws/aws-iot-device-sdk-python#318] [Q&A] [answered]
Hello, I have subscribed to 100 different topics but I am able to get messages from 50 topics only
import json
import time

from AWSIoTPythonSDK.MQTTLib import AWSIoTMQTTClient

def aqi1CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-1.json", "w") as outfile:
        outfile.write(msg)

def aqi2CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-2.json", "w") as outfile:
        outfile.write(msg)

def aqi3CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-3.json", "w") as outfile:
        outfile.write(msg)

def aqi4CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-4.json", "w") as outfile:
        outfile.write(msg)

def aqi5CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-5.json", "w") as outfile:
        outfile.write(msg)

def aqi6CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-6.json", "w") as outfile:
        outfile.write(msg)

def aqi7CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-7.json", "w") as outfile:
        outfile.write(msg)

def aqi8CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-8.json", "w") as outfile:
        outfile.write(msg)

def aqi9CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-9.json", "w") as outfile:
        outfile.write(msg)

def aqi10CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-10.json", "w") as outfile:
        outfile.write(msg)

def aqi11CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-11.json", "w") as outfile:
        outfile.write(msg)

def aqi12CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-12.json", "w") as outfile:
        outfile.write(msg)

def aqi13CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-13.json", "w") as outfile:
        outfile.write(msg)

def aqi14CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-14.json", "w") as outfile:
        outfile.write(msg)

def aqi15CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-15.json", "w") as outfile:
        outfile.write(msg)

def aqi16CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-16.json", "w") as outfile:
        outfile.write(msg)

def aqi17CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-17.json", "w") as outfile:
        outfile.write(msg)

def aqi18CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-18.json", "w") as outfile:
        outfile.write(msg)

def aqi19CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-19.json", "w") as outfile:
        outfile.write(msg)

def aqi20CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-20.json", "w") as outfile:
        outfile.write(msg)

def aqi21CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-21.json", "w") as outfile:
        outfile.write(msg)

def aqi22CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-22.json", "w") as outfile:
        outfile.write(msg)

def aqi23CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-23.json", "w") as outfile:
        outfile.write(msg)

def aqi24CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-24.json", "w") as outfile:
        outfile.write(msg)

def aqi25CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-25.json", "w") as outfile:
        outfile.write(msg)

def aqi26CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-26.json", "w") as outfile:
        outfile.write(msg)

def aqi27CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-27.json", "w") as outfile:
        outfile.write(msg)

def aqi28CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-28.json", "w") as outfile:
        outfile.write(msg)

def aqi29CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-29.json", "w") as outfile:
        outfile.write(msg)

def aqi30CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-30.json", "w") as outfile:
        outfile.write(msg)

def aqi31CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-31.json", "w") as outfile:
        outfile.write(msg)

def aqi32CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-32.json", "w") as outfile:
        outfile.write(msg)

def aqi33CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-33.json", "w") as outfile:
        outfile.write(msg)

def aqi34CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-34.json", "w") as outfile:
        outfile.write(msg)

def aqi35CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-35.json", "w") as outfile:
        outfile.write(msg)

def aqi36CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-36.json", "w") as outfile:
        outfile.write(msg)

def aqi37CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-37.json", "w") as outfile:
        outfile.write(msg)

def aqi38CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-38.json", "w") as outfile:
        outfile.write(msg)

def aqi39CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-39.json", "w") as outfile:
        outfile.write(msg)

def aqi40CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-40.json", "w") as outfile:
        outfile.write(msg)

def aqi41CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-41.json", "w") as outfile:
        outfile.write(msg)

def aqi42CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-42.json", "w") as outfile:
        outfile.write(msg)

def aqi43CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-43.json", "w") as outfile:
        outfile.write(msg)

def aqi44CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-44.json", "w") as outfile:
        outfile.write(msg)

def aqi45CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-45.json", "w") as outfile:
        outfile.write(msg)

def aqi46CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-46.json", "w") as outfile:
        outfile.write(msg)

def aqi47CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-47.json", "w") as outfile:
        outfile.write(msg)

def aqi48CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-48.json", "w") as outfile:
        outfile.write(msg)

def aqi49CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-49.json", "w") as outfile:
        outfile.write(msg)

def aqi50CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-50.json", "w") as outfile:
        outfile.write(msg)

def aqi51CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-51.json", "w") as outfile:
        outfile.write(msg)

def aqi52CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-52.json", "w") as outfile:
        outfile.write(msg)

def aqi53CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-53.json", "w") as outfile:
        outfile.write(msg)

def aqi54CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-54.json", "w") as outfile:
        outfile.write(msg)

def aqi55CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-55.json", "w") as outfile:
        outfile.write(msg)

def aqi56CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-56.json", "w") as outfile:
        outfile.write(msg)

def aqi57CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-57.json", "w") as outfile:
        outfile.write(msg)

def aqi58CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi_58.json", "w") as outfile:
        outfile.write(msg)

def aqi59CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-59.json", "w") as outfile:
        outfile.write(msg)

def aqi60CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-60.json", "w") as outfile:
        outfile.write(msg)

def aqi61CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-61.json", "w") as outfile:
        outfile.write(msg)

def aqi62CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-62.json", "w") as outfile:
        outfile.write(msg)

def aqi63CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-63.json", "w") as outfile:
        outfile.write(msg)

def aqi64CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-64.json", "w") as outfile:
        outfile.write(msg)

def aqi65CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-65.json", "w") as outfile:
        outfile.write(msg)

def aqi66CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-66.json", "w") as outfile:
        outfile.write(msg)

def aqi67CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-67.json", "w") as outfile:
        outfile.write(msg)

def aqi68CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-68.json", "w") as outfile:
        outfile.write(msg)

def aqi69CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-69.json", "w") as outfile:
        outfile.write(msg)

def aqi70CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-70.json", "w") as outfile:
        outfile.write(msg)

def aqi71CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-71.json", "w") as outfile:
        outfile.write(msg)

def aqi72CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-72.json", "w") as outfile:
        outfile.write(msg)

def aqi73CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-73.json", "w") as outfile:
        outfile.write(msg)

def aqi74CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-74.json", "w") as outfile:
        outfile.write(msg)

def aqi75CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-75.json", "w") as outfile:
        outfile.write(msg)

def aqi76CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-76.json", "w") as outfile:
        outfile.write(msg)

def aqi77CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-77.json", "w") as outfile:
        outfile.write(msg)

def aqi78CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-78.json", "w") as outfile:
        outfile.write(msg)

def aqi79CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-79.json", "w") as outfile:
        outfile.write(msg)

def aqi80CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-80.json", "w") as outfile:
        outfile.write(msg)

def aqi81CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-81.json", "w") as outfile:
        outfile.write(msg)

def aqi82CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-82.json", "w") as outfile:
        outfile.write(msg)

def aqi83CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-83.json", "w") as outfile:
        outfile.write(msg)

def aqi84CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-84.json", "w") as outfile:
        outfile.write(msg)

def aqi85CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-85.json", "w") as outfile:
        outfile.write(msg)

def aqi86CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-86.json", "w") as outfile:
        outfile.write(msg)

def aqi87CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-87.json", "w") as outfile:
        outfile.write(msg)

def aqi88CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-88.json", "w") as outfile:
        outfile.write(msg)

def aqi89CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-89.json", "w") as outfile:
        outfile.write(msg)

def aqi90CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-90.json", "w") as outfile:
        outfile.write(msg)

def aqi91CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-91.json", "w") as outfile:
        outfile.write(msg)

def aqi92CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-92.json", "w") as outfile:
        outfile.write(msg)

def aqi93CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-93.json", "w") as outfile:
        outfile.write(msg)

def aqi94CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-94.json", "w") as outfile:
        outfile.write(msg)

def aqi95CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-95.json", "w") as outfile:
        outfile.write(msg)

def aqi96CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-96.json", "w") as outfile:
        outfile.write(msg)

def aqi97CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-97.json", "w") as outfile:
        outfile.write(msg)

def aqi98CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-98.json", "w") as outfile:
        outfile.write(msg)

def aqi99CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-99.json", "w") as outfile:
        outfile.write(msg)

def aqi100CallBack(client, userdata, message):
    data = message.payload
    msg = data.decode()
    print(msg)
    #jsonmsg =json.loads(data)
    with open("aqi-100.json", "w") as outfile:
        outfile.write(msg)

myMQTTClient = AWSIoTMQTTClient("368032")
myMQTTClient.configureEndpoint(
    "--------------------.us-east-1.amazonaws.com", 8883)
myMQTTClient.configureCredentials(
    "./AmazonRootCA1.pem", "./private.pem.key", "./certificate.pem.crt")

myMQTTClient.connect()
print("client connected")

while True:

    myMQTTClient.subscribe("aqi-1", 1, aqi1CallBack)
    myMQTTClient.subscribe("aqi-2", 1, aqi2CallBack)
    myMQTTClient.subscribe("aqi-3", 1, aqi3CallBack)
    myMQTTClient.subscribe("aqi-4", 1, aqi4CallBack)
    myMQTTClient.subscribe("aqi-5", 1, aqi5CallBack)
    myMQTTClient.subscribe("aqi-6", 1, aqi6CallBack)
    myMQTTClient.subscribe("aqi-7", 1, aqi7CallBack)
    myMQTTClient.subscribe("aqi-8", 1, aqi8CallBack)
    myMQTTClient.subscribe("aqi-9", 1, aqi9CallBack)
    myMQTTClient.subscribe("aqi-10", 1, aqi10CallBack)

    myMQTTClient.subscribe("aqi-11", 1, aqi11CallBack)
    myMQTTClient.subscribe("aqi-12", 1, aqi12CallBack)
    myMQTTClient.subscribe("aqi-13", 1, aqi13CallBack)
    myMQTTClient.subscribe("aqi-14", 1, aqi14CallBack)
    myMQTTClient.subscribe("aqi-15", 1, aqi15CallBack)
    myMQTTClient.subscribe("aqi-16", 1, aqi16CallBack)
    myMQTTClient.subscribe("aqi-17", 1, aqi17CallBack)
    myMQTTClient.subscribe("aqi-18", 1, aqi18CallBack)
    myMQTTClient.subscribe("aqi-19", 1, aqi19CallBack)
    myMQTTClient.subscribe("aqi-20", 1, aqi20CallBack)

    myMQTTClient.subscribe("aqi-21", 1, aqi21CallBack)
    myMQTTClient.subscribe("aqi-22", 1, aqi22CallBack)
    myMQTTClient.subscribe("aqi-23", 1, aqi23CallBack)
    myMQTTClient.subscribe("aqi-24", 1, aqi24CallBack)
    myMQTTClient.subscribe("aqi-25", 1, aqi25CallBack)
    myMQTTClient.subscribe("aqi-26", 1, aqi26CallBack)
    myMQTTClient.subscribe("aqi-27", 1, aqi27CallBack)
    myMQTTClient.subscribe("aqi-28", 1, aqi28CallBack)
    myMQTTClient.subscribe("aqi-29", 1, aqi29CallBack)
    myMQTTClient.subscribe("aqi-30", 1, aqi30CallBack)

    myMQTTClient.subscribe("aqi-31", 1, aqi31CallBack)
    myMQTTClient.subscribe("aqi-32", 1, aqi32CallBack)
    myMQTTClient.subscribe("aqi-33", 1, aqi33CallBack)
    myMQTTClient.subscribe("aqi-34", 1, aqi34CallBack)
    myMQTTClient.subscribe("aqi-35", 1, aqi35CallBack)
    myMQTTClient.subscribe("aqi-36", 1, aqi36CallBack)
    myMQTTClient.subscribe("aqi-37", 1, aqi37CallBack)
    myMQTTClient.subscribe("aqi-38", 1, aqi38CallBack)
    myMQTTClient.subscribe("aqi-39", 1, aqi39CallBack)
    myMQTTClient.subscribe("aqi-40", 1, aqi40CallBack)

    myMQTTClient.subscribe("aqi-41", 1, aqi41CallBack)
    myMQTTClient.subscribe("aqi-42", 1, aqi42CallBack)
    myMQTTClient.subscribe("aqi-43", 1, aqi43CallBack)
    myMQTTClient.subscribe("aqi-44", 1, aqi44CallBack)
    myMQTTClient.subscribe("aqi-45", 1, aqi45CallBack)
    myMQTTClient.subscribe("aqi-46", 1, aqi46CallBack)
    myMQTTClient.subscribe("aqi-47", 1, aqi47CallBack)
    myMQTTClient.subscribe("aqi-48", 1, aqi48CallBack)
    myMQTTClient.subscribe("aqi-49", 1, aqi49CallBack)
    myMQTTClient.subscribe("aqi-50", 1, aqi50CallBack)

    myMQTTClient.subscribe("aqi-51", 1, aqi51CallBack)
    myMQTTClient.subscribe("aqi-52", 1, aqi52CallBack)
    myMQTTClient.subscribe("aqi-53", 1, aqi53CallBack)
    myMQTTClient.subscribe("aqi-54", 1, aqi54CallBack)
    myMQTTClient.subscribe("aqi-55", 1, aqi55CallBack)
    myMQTTClient.subscribe("aqi-56", 1, aqi56CallBack)
    myMQTTClient.subscribe("aqi-57", 1, aqi57CallBack)
    myMQTTClient.subscribe("aqi-58", 1, aqi58CallBack)
    myMQTTClient.subscribe("aqi-59", 1, aqi59CallBack)
    myMQTTClient.subscribe("aqi-60", 1, aqi60CallBack)

    myMQTTClient.subscribe("aqi-61", 1, aqi61CallBack)
    myMQTTClient.subscribe("aqi-62", 1, aqi62CallBack)
    myMQTTClient.subscribe("aqi-63", 1, aqi63CallBack)
    myMQTTClient.subscribe("aqi-64", 1, aqi64CallBack)
    myMQTTClient.subscribe("aqi-65", 1, aqi65CallBack)
    myMQTTClient.subscribe("aqi-66", 1, aqi66CallBack)
    myMQTTClient.subscribe("aqi-67", 1, aqi67CallBack)
    myMQTTClient.subscribe("aqi-68", 1, aqi68CallBack)
    myMQTTClient.subscribe("aqi-69", 1, aqi69CallBack)
    myMQTTClient.subscribe("aqi-70", 1, aqi70CallBack)

    myMQTTClient.subscribe("aqi-71", 1, aqi71CallBack)
    myMQTTClient.subscribe("aqi-72", 1, aqi72CallBack)
    myMQTTClient.subscribe("aqi-73", 1, aqi73CallBack)
    myMQTTClient.subscribe("aqi-74", 1, aqi74CallBack)
    myMQTTClient.subscribe("aqi-75", 1, aqi75CallBack)
    myMQTTClient.subscribe("aqi-76", 1, aqi76CallBack)
    myMQTTClient.subscribe("aqi-77", 1, aqi77CallBack)
    myMQTTClient.subscribe("aqi-78", 1, aqi78CallBack)
    myMQTTClient.subscribe("aqi-79", 1, aqi79CallBack)
    myMQTTClient.subscribe("aqi-80", 1, aqi80CallBack)

    myMQTTClient.subscribe("aqi-81", 1, aqi81CallBack)
    myMQTTClient.subscribe("aqi-82", 1, aqi82CallBack)
    myMQTTClient.subscribe("aqi-83", 1, aqi83CallBack)
    myMQTTClient.subscribe("aqi-84", 1, aqi84CallBack)
    myMQTTClient.subscribe("aqi-85", 1, aqi85CallBack)
    myMQTTClient.subscribe("aqi-86", 1, aqi86CallBack)
    myMQTTClient.subscribe("aqi-87", 1, aqi87CallBack)
    myMQTTClient.subscribe("aqi-88", 1, aqi88CallBack)
    myMQTTClient.subscribe("aqi-89", 1, aqi89CallBack)
    myMQTTClient.subscribe("aqi-90", 1, aqi90CallBack)

    myMQTTClient.subscribe("aqi-91", 1, aqi91CallBack)
    myMQTTClient.subscribe("aqi-92", 1, aqi92CallBack)
    myMQTTClient.subscribe("aqi-93", 1, aqi93CallBack)
    myMQTTClient.subscribe("aqi-94", 1, aqi94CallBack)
    myMQTTClient.subscribe("aqi-95", 1, aqi94CallBack)
    myMQTTClient.subscribe("aqi-96", 1, aqi96CallBack)
    myMQTTClient.subscribe("aqi-97", 1, aqi97CallBack)
    myMQTTClient.subscribe("aqi-98", 1, aqi98CallBack)
    myMQTTClient.subscribe("aqi-9",  1, aqi99CallBack)
    myMQTTClient.subscribe("aqi-100", 1, aqi100CallBack)
    )

    x = input()

    myMQTTClient.unsubscribe("tankMonitoringOne/pub")

    myMQTTClient.disconnect()
    print("client disconnected")

**Comment by bretambrose** [comment, marked as answer]
IoT Core has a 50 subscription per connection limit: https://docs.aws.amazon.com/general/latest/gr/iot-core.html#message-broker-limits
